application.go

```diff
@@ -130,6 +131,17 @@ type DeleteAppOpts struct {
 	Force bool `url:"force,omitempty"`
 }
 
+// TaskStats is a container for Stats
+type TaskStats struct {
+	Stats Stats `json:"stats"`
+}
+
+// Stats is a collection of aggregate statistics about an application's tasks
+type Stats struct {
+	Counts   map[string]int     `json:"counts"`
+	LifeTime map[string]float64 `json:"lifetime"`
```

Comment:
If the JSON field name uses a capital T, then I agree we should have it in the variable name as well. You did not use a capital T in the JSON tag name though -- is that a typo?

Whatever the right spelling is, we should keep it in sync.